torch_xla/_internal/rendezvous.py

```diff
@@ -44,4 +43,8 @@ def pjrt_rendezvous_handler(url: str,
             xr.process_count(),
             is_master=xr.process_index() == 0)
 
-  yield (_store, xr.global_ordinal(), world_size)
+  # In SPMD, the world size and rank are determined by the process count and
```

Comment:
In both cases the world size and rank should be determined by the process count and index -- this is needed due to the fact that `xr.world_size()` returns 1 for a single-replica SPMD case. I think we could instead explain this, saying that "In SPMD we use the process count and index directly, since the xr.world_size() returns 1 without any replication group set." Or you can also add some more in the context of the CPU process group need for SPMD?